### DISCUSSION TITLE
Add Pixel 5/5a, 6/Pro/a, 7/Pro/a

### DISCUSSION THREAD

**Discussion by tejasraman** [Genymobile/feedback#64] [Genymotion Desktop] [answered]
The current templates for Pixel devices are quite outdated. Would it be possible to add templates for these modern Pixel devices?
- Pixel 5
- Pixel 5a
- Pixel 6 Pro (not requesting 6 as it is too similar to 6 Pro)
- Pixel 6a
- Pixel 7
- Pixel 7a

**Comment by agavignet** [comment, marked as answer]
Hi @tejasraman,

Thanks for your patience, we added the templates you suggested, hope you will enjoy them.

Best regards

**Reply by tejasraman** [reply]
Thanks so much @agavignet - I wanted them to screen record byt ended up buying an actual Pixel as I needed full GMS and PixelLauncher, but thanks anyway! :)
